rskj-core/src/test/java/co/rsk/rpc/modules/eth/EthModuleTest.java

```diff
@@ -78,6 +80,50 @@ public void callSmokeTest() {
         assertThat(result, is(TypeConverter.toJsonHex(hreturn)));
     }
 
+    @Test
+    public void test_revertedTransaction() {
+        Web3.CallArguments args = new Web3.CallArguments();
+        BlockResult blockResult = mock(BlockResult.class);
+        Block block = mock(Block.class);
+        ExecutionBlockRetriever retriever = mock(ExecutionBlockRetriever.class);
+        when(retriever.getExecutionBlock_workaround("latest"))
+                .thenReturn(blockResult);
+        when(blockResult.getBlock()).thenReturn(block);
+
+        byte[] hreturn = Hex.decode(
+                "08c379a000000000000000000000000000000000000000000000000000000000" +
+                        "0000002000000000000000000000000000000000000000000000000000000000" +
+                        "0000000f6465706f73697420746f6f2062696700000000000000000000000000" +
+                        "00000000");
```

Comment:
What does this value means? Add some comments. We could check other possible values (?)